hs_geographic_feature_resource/receivers.py

```diff
@@ -0,0 +1,522 @@
+__author__ = 'drew'
+
+
+from django.dispatch import receiver
+from hs_core.signals import *
+from hs_geographic_feature_resource.models import GeographicFeatureResource, OriginalCoverage, GeographicFeatureMetaData
+from hs_core import hydroshare
+import tempfile
+import shutil
+import os
+import pprint
+from hs_geographic_feature_resource.parse_lib import *
+from hs_geographic_feature_resource.forms import *
+import zipfile, json
+from hs_core.hydroshare.resource import ResourceFile
```

Comment:
Please organize these imports grouped by Python standard library imports, Django imports, 3rd party libraries, and finally Hydroshare libraries at the end. Provide spacing between each group and try to sort them logically or alphabetically.